src/Command/Processor/KeyPrefixProcessor.php

```diff
@@ -406,6 +409,23 @@ public static function evalKeys(CommandInterface $command, $prefix)
         }
     }
 
+    /**
+     * Applies the specified prefix to the keys of an FCall-based command.
+     *
+     * @param CommandInterface $command Command instance.
+     * @param string           $prefix  Prefix string.
+     */
+    public static function fCallKeys(CommandInterface $command, $prefix)
```

Comment:
This code is identical to `evalKeys()`. Can you use that function, or at least call it from `fCallKeys()`?

Reply:
Yes I know, since it's name is `evalKeys`, I just thought to add a new function name, if you prefer to use `evalKeys` for both `EVAL` and `FCALL`, then I will change it.

Reply:
Yes I know, since it's name is `evalKeys`, I just thought to add a new function name, if you prefer to use `evalKeys` for both `EVAL` and `FCALL`, then I will change it.

Reply:
Yes, either reuse the function, or call it from within it, instead of duplicate code.